rpcclient/infrastructure.go

```diff
@@ -90,6 +91,12 @@ const (
 	// requestRetryInterval is the initial amount of time to wait in between
 	// retries when sending HTTP POST requests.
 	requestRetryInterval = time.Millisecond * 500
+
+	// rpcRequestInterval is the duration to wait before making more RPC
+	// requests to bitcoind. This is only used by low priority messages.
+	// For every 4 messages, we would sleep this duration before making
+	// more post requests.
+	rpcRequestInterval = time.Millisecond * 10
```

Comment:
Could we maybe make this configurable when instantiating the client?

Reply:
Done!